projects/storefrontlib/src/cms-components/checkout/components/review-submit/review-submit.component.ts

```diff
@@ -192,12 +194,32 @@ export class ReviewSubmitComponent {
         month: paymentDetails.expiryMonth,
         year: paymentDetails.expiryYear,
       }),
+      this.translation.translate('paymentForm.billingAddress'),
     ]).pipe(
-      map(([textTitle, textExpires]) => {
+      map(([textTitle, textExpires, billingAddress]) => {
+        const region = paymentDetails.billingAddress?.region?.isocode
```

Comment:
this becomes undefined when changing from card > account > back to card

For example, card until review page, then go back to the first step. Repeat for account, and go back to first step. Finally, choose card again until review page again, and region is undefined until refreshed. Also, contains the billing address!

Lets talk for this one